<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerEMSegmentRunSegmentationStep</class>
 <widget class="qSlicerEMSegmentWorkflowWidgetStep" name="qSlicerEMSegmentRunSegmentationStep">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>578</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="WorkingDirectoryGroupBox">
     <property name="title">
      <string>Working Directory</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="SaveIntermediateResultsCheckBox">
        <property name="text">
         <string>Save Intermediate Results</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkDirectoryButton" name="DirectoryButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="OutputLabelMapGroupBox">
     <property name="title">
      <string>Output Label Map</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="qMRMLNodeComboBox" name="OutputLabelMapComboBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showHidden">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>true</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="editEnabled">
         <bool>false</bool>
        </property>
        <property name="baseName">
         <string>EMSegment</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="DefineVOIGroupBox">
     <property name="title">
      <string>Define Volume of Interest (VOI)</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="qMRMLROIWidget" name="ROIWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Display2DVOILayout" stretch="0,0,0,0">
        <item>
         <spacer name="spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="Display2DVOIButton">
          <property name="icon">
           <iconset resource="../../../../Libs/qMRMLWidgets/Resources/qMRMLWidgets.qrc">
            <normaloff>:/Icons/VisibleOff.png</normaloff>
            <normalon>:/Icons/VisibleOn.png</normalon>:/Icons/VisibleOff.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Display2DVOILabel">
          <property name="text">
           <string>Display VOI in 2D Viewer</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="MiscGroupBox">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QCheckBox" name="MultithreadingCheckBox">
        <property name="text">
         <string>Multi-threading Enabled</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkDirectoryButton</class>
   <extends>QWidget</extends>
   <header>ctkDirectoryButton.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLROIWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLROIWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkWorkflowWidgetStep</class>
   <extends>QWidget</extends>
   <header>ctkWorkflowWidgetStep.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerEMSegmentWorkflowWidgetStep</class>
   <extends>ctkWorkflowWidgetStep</extends>
   <header>qSlicerEMSegmentWorkflowWidgetStep.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../Libs/qMRMLWidgets/Resources/qMRMLWidgets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>SaveIntermediateResultsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>DirectoryButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerEMSegmentRunSegmentationStep</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>OutputLabelMapComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>4</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
